File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1080_basico.js
import { question } from "readline-sync";

function main() {
    let limite_min = 1;
    let limite_max = 100;
    let maior = 0;
    let posicao = 0

    while (limite_min <= limite_max) {
        let numero = get_number();

        if (numero > maior) {
            maior = numero;
            posicao = limite_min;
        }

        limite_min++;
    }
    print(`${maior}`);
    print(`${posicao}`);
}

function get_number() {
    return parseInt(question());
}

function print(texto) {
    return console.log(texto);
}

main();

File: Exercicios_ED_I/Bee/1080_basico.cpp
#include <stdio.h>
#include <iostream>
using namespace std;

main() {
    int i = 0;
    int maior = 0;
    int posicao = 0;

    while (i < 100) {
        int numero;
        cin >> numero;

        if (numero > maior) {
            maior = numero;
            posicao = i + 1;
        }
        i++;
    }

    cout << maior << endl;
    cout << posicao << endl;
}